wiki/People/The_Team/es.md

```diff
@@ -1,75 +1,96 @@
 ---
-outdated: true
+tags:
+  - osu! staff
+  - osu!team
+  - osu! team
+  - staff
+  - team
 ---
 
 # El Equipo
 
-Las personas que nombradas a continuación son el núcleo del equipo de osu!, y son los principales responsables de ayudar a que el juego sea impresionante para todos.
+*Para una lista de promociones y retiros, vea: [Staff Log](/wiki/Staff_Log)*
 
-## Creador
+Las personas nombradas a continuación son el núcleo del equipo de osu!, y son los principales responsables de ayudar a que el juego sea impresionante para todos.
```

Comment:
should say something about how this is called "osu!team"

Reply:
It is already there, is just that I think that "equipo de osu!" fits better than "osu!equipo".
Let me now if you want me to change it to "osu!equipo".